packages/extension/src/ui/useEntryRoute.tsx

```diff
@@ -37,6 +37,10 @@ const determineEntry = async (query: URLSearchParams) => {
     return routes.ledgerEntry()
   }
 
+  if (query.get("goto") === "/multisig/create") {
```

Comment:
Let's keep this simple and consistent with ledger above

```suggestion
  if (query.get("goto") === "multisig") {
```